### PR TITLE
feat: make contract fee rate configurable

### LINKED ISSUE #628: iOS status bar is not visible
We obscure the status bar, both on light and dark system theme.

IMG_F4420D64634C-1

Probably the best way is changing the background to purple (or a little lighter colour).

### PR DESCRIPTION
**Pull request by bonomat**
While we can't update existing contracts, we can at least change the settings when needed for new contracts.

Resolves https://github.com/get10101/10101/issues/1159
Resolves #628 

image

### CODE REVIEW

File: mobile/ios/Podfile.lock [hunk 1]
```diff
@@ -1,6 +1,4 @@
 PODS:
```
Comment (holzeis) [inline mobile/ios/Podfile.lock:1]: 🙃 You could have put this into a separate commit

**Review by holzeis — APPROVED** [review]
LGTM! Thanks 👍

File: mobile/ios/Podfile.lock (hunk 1 above)
Comment (bonomat) [reply mobile/ios/Podfile.lock:1]: 🙈
